docs/release-notes/index.rst

```diff
@@ -0,0 +1,257 @@
+.. _release-notes:
+
+Release notes
+=============
+
+This page lists past release notes for Rockcraft, summarising new features, bug
+fixes and backwards-incompatible changes in each version. It also contains the
+release and support policies for Rockcraft.
+
+
+Current releases
+----------------
+
+
+Rockcraft 1
+~~~~~~~~~~~
+
+- :ref:`release-1.6.0`.
+
+.. _release_policy_and_schedule:
+
+Release policy and schedule
+---------------------------
+
+Canonical is committed to releasing Rockcraft on a regular monthly
+cadence. Our goal is to have new features and fixes available as soon as
+possible. On the first Monday of the month, we publish a major or minor
+version of Rockcraft to the candidate channel, where it undergoes testing and
+feedback. If we determine the candidate is ready for mainstream use by the
+Monday of the week after, we release it as a stable version. During testing
+and feedback of a candidate, we prioritise fixes for critical issues.
+
+Rockcraft release naming follows the Semantic Versioning 2.0.0 scheme with
+numbers for major, minor, and patch versions.
+
+.. list-table::
+    :header-rows: 1
+
+    * - Version
+      - Example
+      - Significance
+    * - Major
+      - **3**.1.2
+      - A change that drops support for an earlier software base.
+    * - Minor
+      - 3.\ **1**\ .2
+      - A new feature within the major version.
+    * - Patch
+      - 3.1.\ **2**
+      - A bug fix within the major or minor version.
+
+
+Long-term support
+-----------------
+
+Rockcraft doesn't have long-term support (LTS) releases. However, we typically
+deliver a compatibility release shortly after Ubuntu LTS releases to ensure
+continuity.
+
+Rockcraft software bases are derived from Ubuntu LTS releases, and their
+development keeps pace with the OS's new releases and support lifecycle.
+
+.. toctree::
+   :maxdepth: 1
+
+
+.. release note template:
+
+  Starcraft 2.0 release notes
```

Comment:
Just to clarify, are these notes for individual releases meant to be added as separate files under `release-notes/`?

Reply:
Correct, yes. So there should be a file for Rockcraft 1.6.0 in this dir. Best to name it `rockcraft-1-6-0`.

Reply:
Oh, I see, I missed your second point about splitting this into two PRs.

For now this is fine, but let's make sure in the next PR that this index no longer links to the changelog.

Reply:
I don't think we need to write notes for 1.6.0. Let's just focus on making a good example using 1.7.0.

Reply:
updated (removed the link to the 1.6.0 changelog)